tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/GitHelper.cs

```diff
@@ -104,10 +118,26 @@ public string DiscoverRepoRoot(string path)
             return gitDir.Parent?.FullName ?? throw new InvalidOperationException("Unable to determine repository root");
         }
 
+        // Get the repository name from the local path
         public string GetRepoName(string path)
         {
-            var repoRoot = DiscoverRepoRoot(path);
-            return new DirectoryInfo(repoRoot).Name ?? throw new InvalidOperationException($"Unable to determine repository name for path: {path}");
+            if (!string.IsNullOrEmpty(path))
```

Comment:
Nit: for code like this I think it's cleaner to invert the conditional. That way the method body isn't in a nested block (making long lines harder to read, etc.) and the reader can easily find the error/validation conditions defined at the top alongside the signature.

```
if (string.IsNullOrEmpty(path))
{
    throw new ArgumentException("Invalid repository path.", nameof(path));
}

... remaining method body ...
```